gradle.properties

```diff
@@ -8,9 +8,9 @@ sizeofVersion = 0.3.0
 ehcache2Version = 2.10.3
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.4.0-pre6
-terracottaApisVersion = 1.4.0-pre7
-terracottaCoreVersion = 5.4.0-pre13
+terracottaPlatformVersion = 5.4.0-pre8
```

Comment:
this version needs to be released (https://github.com/Terracotta-OSS/terracotta-platform/pull/420)